apps/applicant-tracking/src/services/api/applicant/index.ts

```diff
@@ -0,0 +1,18 @@
+import { urls } from '@/utils/constants/urls';
+import { zApplicantResponse } from '@shared/types';
+
+class ApiApplicantService {
+  async get(id: string) {
+    const response = await fetch(urls.api.getApplicant(id));
+    const data = await response.json();
+
+    const validationResult = zApplicantResponse.safeParse(data);
```

Comment:
I thought we discussed not validating the db response in the API and letting the frontend do that?